recipes/opentelemetry-proto/all/conanfile.py

```diff
@@ -1,43 +1,44 @@
+import os
+
 from conan import ConanFile
-from conan.tools.files import get, copy, save
+from conan.tools.files import get, copy
 from conan.tools.layout import basic_layout
-import os
 
 required_conan_version = ">=1.52.0"
 
+
 class OpenTelemetryProtoConan(ConanFile):
     name = "opentelemetry-proto"
     license = "Apache-2.0"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/open-telemetry/opentelemetry-proto"
     description = "Protobuf definitions for the OpenTelemetry protocol (OTLP)"
-    topics = ("opentelemetry", "telemetry", "otlp")
+    topics = ("opentelemetry", "telemetry", "otlp", "pre-built")
+
+    package_type = "application"
```

Comment:
```suggestion
    package_type = "build-scripts"
```

https://docs.conan.io/2/reference/conanfile/attributes.html#package-type

As this recipe is packaging only raw file, script fits better due the trait `visible=False`.